target/linux/ramips/image/Makefile

```diff
@@ -79,6 +79,18 @@ define Build/mkdlinkfw
 		-s $(DLINK_FIRMWARE_SIZE)
 endef
 
+define Build/mkdlinkfw-loader
```

Comment:
Should be moved after mkdlinkfw-factory due to sorting?

Reply:
Done.